### PR TITLE
Update locate command to support validated heartbeats and speedtest a…

### PR DESCRIPTION
**Pull request by bbalser**
…verages

### CODE REVIEW

File: file_store/src/heartbeat.rs [hunk 1]
```diff
@@ -76,6 +79,51 @@ impl MsgTimestamp<Result<DateTime<Utc>>> for CellHeartbeatIngestReportV1 {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ValidatedHeartbeat {
```
Comment (andymck) [inline file_store/src/heartbeat.rs:83]: NOTE, there is an existing ValidatedHeartbeat struct in mobile verifier...which does already include all the data you require here.  Have you considered if these can be harmonized ?

Comment (bbalser) [reply file_store/src/heartbeat.rs:83]: missed that those exist, will look at pulling those into file_store

File: file_store/src/heartbeat.rs [hunk 2]
```diff
@@ -76,6 +79,51 @@ impl MsgTimestamp<Result<DateTime<Utc>>> for CellHeartbeatIngestReportV1 {
...
+    pub cbsd_id: String,
+    pub pub_key: PublicKeyBinary,
+    pub reward_multiplier: f32,
```
Comment (jeffgrunewald) [inline file_store/src/heartbeat.rs:86]: should we be keeping all the numbers that factor into rewards calculations be kept as integers or decimals? we've allowed a _few_ floats but I think those have so far always been relative to things like transmit values and other RF-things

**Review by jeffgrunewald — COMMENTED** [review]
assuming you want to remove the similar ValidatedHeartbeat type from the mobile verifier in a separate change

**Review by andymck — APPROVED** [review]

**Comment by bbalser** [comment]
> assuming you want to remove the similar ValidatedHeartbeat type from the mobile verifier in a separate change

They are different enough, I am not sure they should be combined. This new one is meant as the rust version of the proto, where the one in the mobile verifier is the struct representing a validated heartbeat during the validation process and has other stuff in needed for that process.  For the moment, I moved these new ones into a cli module.

**Review by jeffgrunewald — APPROVED** [review]

**Pull request merged**